source/appModules/searchhost.py

```diff
@@ -0,0 +1,10 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2021 NV Access Limited, Joseph Lee
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+"""Windows Search app module for Windows 11 (build 22000) and later.
+"""
+
+# Flake8/F403: this is an alias for SearchUI app module.
+from .searchui import * # NOQA
```

Comment:
Assuming that these apps may diverge eventually, I think it is better to explicit. [See the recently started guide for imports, which doesn't perfectly cover this case](https://github.com/nvaccess/nvda/blob/master/devDocs/codingStandards.md#imports).

```suggestion
from .searchui import AppModule, StartMenuSearchField
__all__ = ["AppModule", "StartMenuSearchField"]
```